# **📊 1: How many armor pieces are available?**

In [0]:
%sql
SELECT COUNT(*) AS total_armors FROM eldenringcatalog.gold.dim_armors;

total_armors 723

In [0]:
%sql
SELECT COUNT(DISTINCT armor_name) AS unique_names FROM eldenringcatalog.gold.dim_armors;

unique_names 723

In [0]:
%sql
SELECT armor_type, COUNT(*) as count FROM eldenringcatalog.gold.dim_armors GROUP BY armor_type;

armor_type count helm 212 chest armor 251 gauntlets 123 leg armor 137

**723 items distributed across four main categories. Chest armor is the most represented type with 251 items, followed by helms with 212 items. Leg armor includes 137 items, while gauntlets make up the smallest category with 123 items. Overall, the distribution shows a balanced selection of armor pieces, with a slightly higher focus on core protective gear such as chest armor and helms.**

# **📊 2: Armor types distribution**

In [0]:
%sql
SELECT 
    armor_type,
    COUNT(*) AS piece_count,
    ROUND(AVG(weight), 2) AS avg_weight
FROM eldenringcatalog.gold.dim_armors
GROUP BY armor_type
ORDER BY piece_count DESC;

armor_type piece_count avg_weight chest armor 251 8.91 helm 212 4.3 leg armor 137 5.74 gauntlets 123 3.25

**The armor weights follow a clear pattern based on coverage. Chest armor is the heaviest, averaging 8.91, making it the main source of protection. Leg armor comes next with an average weight of 5.74, followed by helms at 4.3. Gauntlets are the lightest at 3.25, reflecting their smaller size and limited coverage. Overall, weight decreases consistently from larger armor pieces to smaller ones.**

# **📊 3: Load armor stats and analyze physical negation**

In [0]:
%sql
SELECT 
    armor_name,
    armor_type,
    negation_physical,
    negation_strike,
    negation_slash,
    negation_pierce,
    weight
FROM eldenringcatalog.gold.fact_armor_stats
ORDER BY negation_physical DESC
LIMIT 30;

armor_name armor_type negation_physical negation_strike negation_slash negation_pierce weight Bull-Goat Armor chest armor 20.4 20.2 18.3 18.3 26.5 Verdigris Armor chest armor 20.4 17.7 19.7 18.7 25.9 Fire Prelate Armor chest armor 19.2 17.0 18.3 18.3 24.7 Armor of Solitude chest armor 18.7 17.7 18.3 17.5 24.0 Tree Sentinel Armor chest armor 18.7 15.0 18.7 17.5 18.9 Fire Prelate Armor (Altered) chest armor 18.7 16.3 17.5 17.5 23.6 Banished Knight Armor chest armor 18.7 15.0 19.2 17.5 17.5 Radahn's Lion Armor chest armor 18.7 15.0 18.3 17.5 17.5 Veteran's Armor chest armor 18.7 17.0 18.7 17.5 18.9 Radahn's Lion Armor (Altered) chest armor 18.5 14.6 17.7 17.0 16.4 Tree Sentinel Armor (Altered) chest armor 18.5 14.8 18.5 17.3 18.1 Omen Armor chest armor 18.3 15.0 18.7 18.7 23.1 Banished Knight Armor (Altered) chest armor 18.3 14.4 18.7 16.8 16.5 Armor of Solitude (Altered) chest armor 18.3 17.0 17.5 16.8 23.2 Veteran's Armor (Altered) chest armor 18.1 16.8 18.5 17.3 17.2 Crucible Tree Armor chest armor 17.7 13.9 17.5 15.8 15.5 Beast Champion Armor chest armor 17.5 16.3 18.7 18.3 17.5 Lionel's Armor chest armor 17.5 16.3 19.2 19.7 21.2 Crucible Axe Armor chest armor 17.5 13.9 16.8 16.8 15.5 Beast Champion Armor (Altered) chest armor 17.2 15.9 18.5 18.2 16.4 Lionel's Armor (Altered) chest armor 17.1 15.9 18.8 19.3 20.2 Crucible Tree Armor (Altered) chest armor 17.0 13.4 16.0 15.6 14.9 Malformed Dragon Armor chest armor 16.8 15.0 17.5 16.8 16.0 Gaius's Armor chest armor 16.8 15.7 19.7 19.2 20.1 Young Lion's Armor chest armor 16.8 14.4 16.8 16.8 15.0 Crucible Axe Armor (Altered) chest armor 16.8 13.4 16.0 16.0 14.9 Scaled Armor chest armor 16.0 13.9 16.8 16.0 16.0 Royal Knight Armor chest armor 16.0 14.4 17.5 16.8 15.5 Royal Knight Armor (Altered) chest armor 15.8 14.2 17.3 16.6 15.0 Young Lion's Armor (Altered) chest armor 15.3 13.4 15.3 15.3 13.7

**The top 30 armor pieces ranked by physical negation are dominated entirely by heavy chest armor, with values ranging from 20.4 down to 15.3. The Bull-Goat Armor and Verdigris Armor lead with the highest physical negation at 20.4, followed closely by other extremely heavy sets like Fire Prelate, Tree Sentinel, Banished Knight, and Radahn's Lion variants. Across all entries, high physical defense consistently pairs with strong strike, slash, and pierce negation, confirming that the tank-oriented, ultra-heavy chest pieces offer the best overall physical protection in the dataset.**

# **📊 4: Best physical protection per weight ratio**

In [0]:
%sql
SELECT 
    armor_name,
    armor_type,
    negation_physical,
    weight,
    ROUND(negation_physical / weight, 2) AS protection_per_weight
FROM eldenringcatalog.gold.fact_armor_stats
ORDER BY protection_per_weight DESC
LIMIT 30;

armor_name armor_type negation_physical weight protection_per_weight Circlet of Light helm 2.8 1.0 2.8 Ascetic's Loincloth chest armor 6.1 3.2 1.91 Braided Cord Robe chest armor 6.3 3.4 1.85 Goldmask's Rags chest armor 4.2 2.4 1.75 Aristocrat Garb (Altered) chest armor 7.1 4.1 1.73 Old Aristocrat Shoes leg armor 3.4 2.0 1.7 Ascetic's Ankle Guards leg armor 3.4 2.0 1.7 Cloth Garb chest armor 5.3 3.2 1.66 Aristocrat Garb chest armor 7.8 4.9 1.59 Aristocrat Headband helm 1.9 1.2 1.58 Soiled Loincloth leg armor 2.3 1.5 1.53 Gravebird Anklets leg armor 3.0 2.0 1.5 Cloth Trousers leg armor 3.0 2.0 1.5 Traveler's Clothes chest armor 6.1 4.1 1.49 Messmer Soldier Armor (Altered) chest armor 12.1 8.1 1.49 Godskin Apostle Robe chest armor 6.1 4.1 1.49 Lazuli Robe chest armor 6.1 4.1 1.49 Aristocrat Boots leg armor 4.3 2.9 1.48 Malenia's Armor (Altered) chest armor 10.0 6.8 1.47 Fire Knight Armor (Altered) chest armor 11.9 8.4 1.42 Raging Wolf Armor (Altered) chest armor 12.8 9.0 1.42 Malenia's Armor chest armor 10.9 7.7 1.42 Blue Silver Mail Armor (Altered) chest armor 12.1 8.5 1.42 Common Soldier Cloth Armor chest armor 12.4 8.8 1.41 Tree Surcoat chest armor 12.9 9.2 1.4 Eye Surcoat chest armor 12.9 9.2 1.4 Page Garb (Altered) chest armor 6.3 4.5 1.4 Leather Headband helm 1.4 1.0 1.4 Freyja's Armor (Altered) chest armor 12.9 9.3 1.39 Freyja's Armor chest armor 13.5 9.7 1.39

**The best protection-per-weight ratios come almost entirely from lightweight gear, with the Circlet of Light outperforming all armor pieces at an exceptional 2.8 protection per weight. Very light chest pieces like the Ascetic’s Loincloth, Braided Cord Robe, and Goldmask’s Rags also rank highly, offering efficient physical defense despite their low overall stats. Several light leg pieces and helms appear as well, all staying under 3–5 weight. As the list progresses, slightly heavier but still efficient altered chest armors—such as Malenia’s Armor (Altered), Fire Knight Armor (Altered), and Raging Wolf Armor (Altered)—enter with solid ratios around 1.4. Overall, the data clearly shows that light armor provides the best physical protection relative to its weight, making it ideal for mobility-focused builds.**

# **📊 6: Poise per weight ratio**

In [0]:
%sql
SELECT 
    armor_name,
    armor_type,
    resistance_poise,
    weight,
    ROUND(resistance_poise / weight, 2) AS poise_per_weight
FROM eldenringcatalog.gold.fact_armor_stats
ORDER BY poise_per_weight DESC
LIMIT 30;

armor_name armor_type resistance_poise weight poise_per_weight Circlet of Light helm 5.0 1.0 5.0 Ascetic's Loincloth chest armor 10.0 3.2 3.13 Ascetic's Ankle Guards leg armor 6.0 2.0 3.0 Gravebird Anklets leg armor 5.0 2.0 2.5 Aristocrat Garb chest armor 12.0 4.9 2.45 Ansbach's Attire (Altered) chest armor 10.0 4.1 2.44 Battlemage Robe chest armor 10.0 4.1 2.44 Aristocrat Garb (Altered) chest armor 10.0 4.1 2.44 Aristocrat Boots leg armor 7.0 2.9 2.41 Battlemage Legwraps leg armor 6.0 2.5 2.4 Fingerprint Armor (Altered) chest armor 24.0 10.0 2.4 Godskin Noble Trousers leg armor 6.0 2.5 2.4 High Priest Undergarments leg armor 6.0 2.5 2.4 High Priest Robe chest armor 10.0 4.2 2.38 Ansbach's Attire chest armor 12.0 5.1 2.35 Finger Robe chest armor 12.0 5.1 2.35 Mausoleum Knight Armor (Altered) chest armor 25.0 10.8 2.31 Eye Surcoat chest armor 21.0 9.2 2.28 Tree Surcoat chest armor 21.0 9.2 2.28 Fingerprint Armor chest armor 24.0 10.6 2.26 Knight Armor chest armor 24.0 10.6 2.26 Ansbach's Boots leg armor 7.0 3.1 2.26 All-Knowing Armor (Altered) chest armor 21.0 9.3 2.26 Mausoleum Surcoat chest armor 24.0 10.6 2.26 Omenkiller Robe chest armor 16.0 7.1 2.25 Redmane Knight Armor (Altered) chest armor 24.0 10.8 2.22 Messmer Soldier Armor (Altered) chest armor 18.0 8.1 2.22 Godrick Knight Armor (Altered) chest armor 24.0 10.8 2.22 Gelmir Knight Armor (Altered) chest armor 24.0 10.8 2.22 Leyndell Knight Armor (Altered) chest armor 24.0 10.8 2.22

**Poise efficiency is led by extremely lightweight pieces, with the Circlet of Light standing out at a massive 5 poise per weight, far higher than any other armor. Other top performers include the Ascetic’s Ankle Guards and Ascetic’s Loincloth, both offering unusually high poise for their very low weight. Beyond these outliers, many mid-weight chest armors—such as Fingerprint Armor, Mausoleum Knight Armor (Altered), and various Knight or Surcoat variants—deliver strong poise-per-weight ratios around 2.2–2.4, balancing sturdiness with manageable weight. Overall, the data shows that while heavy armor provides high total poise, the most weight-efficient poise comes from specific light or altered pieces, making them excellent choices for players who want poise without committing to full heavy load.**

# **📊 7: Elemental negation analysis (magic, fire, lightning, holy)**

In [0]:
%sql
SELECT 
    armor_name,
    armor_type,
    negation_magic,
    negation_fire,
    negation_lightning,
    negation_holy,
    weight
FROM eldenringcatalog.gold.fact_armor_stats
ORDER BY negation_magic DESC
LIMIT 30;

armor_name armor_type negation_magic negation_fire negation_lightning negation_holy weight Azur's Glintstone Robe chest armor 15.4 12.8 13.0 13.8 7.1 Lusat's Robe chest armor 15.4 12.8 13.3 13.5 7.1 Preceptor's Long Gown chest armor 14.5 13.3 12.8 12.8 6.3 Dryleaf Robe chest armor 14.1 11.9 12.6 14.1 5.1 Gaius's Armor chest armor 14.1 13.8 13.8 13.8 20.1 Royal Knight Armor chest armor 14.1 13.3 12.4 13.0 15.5 Royal Knight Armor (Altered) chest armor 13.9 13.1 12.2 12.8 15.0 Finger Maiden Robe chest armor 13.8 13.1 13.3 13.8 4.6 Lazuli Robe chest armor 13.8 11.9 12.4 12.8 4.1 Dryleaf Robe (Altered) chest armor 13.8 11.4 12.4 13.8 4.1 Queen's Robe chest armor 13.8 12.6 12.8 13.3 5.1 Braided Cord Robe chest armor 13.6 11.9 12.9 14.0 3.4 Banished Knight Armor chest armor 13.5 13.5 13.0 13.3 17.5 Veteran's Armor chest armor 13.5 14.1 12.8 13.3 18.9 Astrologer Robe chest armor 13.5 13.0 13.3 13.0 6.3 Sage Robe chest armor 13.5 12.6 13.0 13.5 5.1 Radahn's Lion Armor chest armor 13.5 14.1 12.6 13.5 17.5 Fire Prelate Armor chest armor 13.5 19.8 13.0 13.3 24.7 Lionel's Armor chest armor 13.5 14.9 13.0 13.5 21.2 Prophet Robe chest armor 13.5 12.6 13.0 13.0 6.3 Raya Lucarian Robe chest armor 13.5 12.6 12.4 12.8 4.1 Verdigris Armor chest armor 13.5 13.5 14.5 13.3 25.9 Scaled Armor chest armor 13.5 14.1 13.0 13.5 16.0 High Page Clothes chest armor 13.5 12.8 13.0 13.2 5.3 Snow Witch Robe chest armor 13.4 13.4 13.2 13.4 5.5 Traveling Maiden Robe chest armor 13.4 12.9 13.1 13.4 4.9 Corhyn's Robe chest armor 13.3 12.8 12.8 13.5 6.3 Bull-Goat Armor chest armor 13.3 13.5 14.9 12.8 26.5 Sanguine Noble Robe chest armor 13.3 11.9 13.0 13.5 5.1 Noble's Traveling Garb chest armor 13.3 13.5 13.0 13.3 6.3

**The strongest magic-negating armors are mostly light to mid-weight robes, led by Azur’s Glintstone Robe and Lusat’s Robe, both offering the highest magic negation at 15.4 while staying relatively light at 7.1 weight. Other top performers like the Preceptor’s Long Gown, Dryleaf Robe, and Queen’s Robe continue this trend, providing high magic, fire, lightning, and holy negation without heavy load penalties. A few heavy armors—such as Gaius’s Armor, Banished Knight Armor, and Lionel’s Armor—appear on the list, showing strong all-around defenses but at much higher weights. Overall, the data shows that robes and lighter chest pieces offer the best magic defense per weight, while heavy armors only match these values by significantly increasing load.**

# **📊 8: Best magic negation armors**

In [0]:
%sql
SELECT 
    a.armor_type,
    a.armor_name,
    a.negation_magic,
    a.weight
FROM eldenringcatalog.gold.fact_armor_stats a
INNER JOIN (
    SELECT armor_type, MAX(negation_magic) as max_magic
    FROM eldenringcatalog.gold.fact_armor_stats
    GROUP BY armor_type
) max_vals ON a.armor_type = max_vals.armor_type AND a.negation_magic = max_vals.max_magic
ORDER BY a.negation_magic DESC;

armor_type armor_name negation_magic weight chest armor Azur's Glintstone Robe 15.4 7.1 chest armor Lusat's Robe 15.4 7.1 leg armor Preceptor's Trousers 8.3 3.9 helm Nox Mirrorhelm 6.7 7.5 gauntlets Preceptor's Gloves 3.6 2.1

**Across all armor categories, the highest magic negation pieces are dominated by lightweight caster gear. Azur's Glintstone Robe and Lusat's Robe tie for the strongest overall protection with 15.4 magic negation at just 7.1 weight, making chest robes the best option for maximizing magical defense. For leg armor, Preceptor's Trousers lead with 8.3 magic negation at 3.9 weight, while the Nox Mirrorhelm provides the best magic negation among helms with 6.7, though at a slightly heavier 7.5 weight. Finally, Preceptor’s Gloves stand out among gauntlets with 3.6 magic negation at only 2.1 weight. Overall, the best magic defense per armor type consistently comes from light, mage-oriented gear, offering strong protection without the burden of heavy load.**

# **📊9: Resistance analysis (immunity, robustness, focus, vitality)**

In [0]:
%sql
SELECT armor_name, armor_type, resistance_immunity, resistance_robustness, resistance_focus, resistance_vitality, weight
FROM eldenringcatalog.gold.fact_armor_stats
ORDER BY resistance_immunity DESC LIMIT 30;

armor_name armor_type resistance_immunity resistance_robustness resistance_focus resistance_vitality weight Mushroom Body chest armor 102.0 24.0 90.0 76.0 5.1 Verdigris Armor chest armor 82.0 97.0 32.0 45.0 25.9 Rotten Gravekeeper Cloak chest armor 76.0 32.0 42.0 42.0 6.3 Thiollier's Garb chest armor 71.0 28.0 63.0 38.0 7.1 Bull-Goat Armor chest armor 71.0 83.0 46.0 55.0 26.5 Guardian Garb (Full Bloom) chest armor 71.0 46.0 57.0 55.0 8.8 Shadow Militiaman Armor chest armor 69.0 29.0 38.0 38.0 7.1 Fire Prelate Armor chest armor 67.0 63.0 90.0 50.0 24.7 White Reed Armor chest armor 67.0 42.0 55.0 50.0 8.3 Thiollier's Garb (Altered) chest armor 67.0 25.0 60.0 35.0 6.3 Omenkiller Robe chest armor 67.0 28.0 46.0 46.0 7.1 Tree Sentinel Armor chest armor 67.0 90.0 42.0 46.0 18.9 Omen Armor chest armor 67.0 63.0 57.0 76.0 23.1 Gaius's Armor chest armor 67.0 83.0 55.0 55.0 20.1 Vulgar Militia Armor chest armor 67.0 35.0 50.0 50.0 7.7 Ronin's Armor chest armor 66.0 48.0 52.0 57.0 8.5 Depraved Perfumer Robe chest armor 65.0 22.0 55.0 80.0 5.0 Rotten Gravekeeper Cloak (Altered) chest armor 64.0 25.0 34.0 34.0 5.1 Night Maiden Armor chest armor 63.0 38.0 55.0 55.0 8.3 Ivory-Draped Tabard chest armor 63.0 42.0 55.0 57.0 8.3 Bloodsoaked Tabard chest armor 63.0 38.0 55.0 50.0 8.3 Depraved Perfumer Robe (Altered) chest armor 63.0 18.0 48.0 63.0 4.0 Nox Monk Armor chest armor 63.0 42.0 50.0 50.0 8.1 Confessor Armor chest armor 63.0 50.0 50.0 55.0 8.3 Fur Raiment chest armor 63.0 42.0 55.0 38.0 5.1 Leather-Draped Tabard chest armor 63.0 42.0 55.0 55.0 8.3 Lord of Blood's Robe chest armor 63.0 23.0 55.0 55.0 8.1 Veteran's Armor chest armor 63.0 90.0 46.0 46.0 18.9 Chain-Draped Tabard chest armor 63.0 42.0 55.0 55.0 8.3 Cleanrot Armor chest armor 63.0 67.0 28.0 32.0 15.0

**The top 30 armors for immunity resistance are almost entirely chest pieces, showing that major body armor dominates this category. The Mushroom Body stands out with an exceptionally high 102 immunity at just 5.1 weight, making it by far the most efficient for resisting status effects. Other notable chest armors include Verdigris Armor (82 immunity, 25.9 weight) and Rotten Gravekeeper Cloak (76 immunity, 6.3 weight), which provide strong protection but vary significantly in weight. Several heavy armors like Bull-Goat Armor, Fire Prelate Armor, and Gaius’s Armor offer high immunity in combination with robust and vitality defenses, though at much higher weights. Overall, the dataset highlights that light special-purpose chest pieces maximize immunity, while heavier armors trade efficiency for all-around durability.**

In [0]:
%sql
SELECT armor_name, armor_type, resistance_immunity, negation_physical, weight FROM eldenringcatalog.gold.fact_armor_stats ORDER BY resistance_immunity DESC LIMIT 20;

armor_name armor_type resistance_immunity negation_physical weight Mushroom Body chest armor 102.0 6.1 5.1 Verdigris Armor chest armor 82.0 20.4 25.9 Rotten Gravekeeper Cloak chest armor 76.0 6.7 6.3 Guardian Garb (Full Bloom) chest armor 71.0 10.2 8.8 Thiollier's Garb chest armor 71.0 8.8 7.1 Bull-Goat Armor chest armor 71.0 20.4 26.5 Shadow Militiaman Armor chest armor 69.0 7.1 7.1 Thiollier's Garb (Altered) chest armor 67.0 8.0 6.3 Tree Sentinel Armor chest armor 67.0 18.7 18.9 Gaius's Armor chest armor 67.0 16.8 20.1 Vulgar Militia Armor chest armor 67.0 8.8 7.7 Omenkiller Robe chest armor 67.0 8.8 7.1 White Reed Armor chest armor 67.0 9.5 8.3 Omen Armor chest armor 67.0 18.3 23.1 Fire Prelate Armor chest armor 67.0 19.2 24.7 Ronin's Armor chest armor 66.0 10.5 8.5 Depraved Perfumer Robe chest armor 65.0 6.0 5.0 Rotten Gravekeeper Cloak (Altered) chest armor 64.0 6.1 5.1 Chain-Draped Tabard chest armor 63.0 9.5 8.3 Cleanrot Armor chest armor 63.0 14.6 15.0

In [0]:
%sql
SELECT armor_name, armor_type, resistance_robustness, negation_physical, weight FROM eldenringcatalog.gold.fact_armor_stats ORDER BY resistance_robustness DESC LIMIT 20;

armor_name armor_type resistance_robustness negation_physical weight Radahn's Lion Armor chest armor 99.0 18.7 17.5 Lionel's Armor chest armor 99.0 17.5 21.2 Verdigris Armor chest armor 97.0 20.4 25.9 Tree Sentinel Armor chest armor 90.0 18.7 18.9 Lionel's Armor (Altered) chest armor 90.0 17.1 20.2 Veteran's Armor chest armor 90.0 18.7 18.9 Beast Champion Armor chest armor 90.0 17.5 17.5 Bull-Goat Armor chest armor 83.0 20.4 26.5 Gaius's Armor chest armor 83.0 16.8 20.1 Scaled Armor chest armor 83.0 16.0 16.0 Banished Knight Armor chest armor 83.0 18.7 17.5 Radahn's Lion Armor (Altered) chest armor 82.0 18.5 16.4 Blaidd's Armor chest armor 76.0 14.6 13.7 Malformed Dragon Armor chest armor 76.0 16.8 16.0 Beast Champion Armor (Altered) chest armor 75.0 17.2 16.4 Tree Sentinel Armor (Altered) chest armor 75.0 18.5 18.1 Veteran's Armor (Altered) chest armor 75.0 18.1 17.2 Briar Armor chest armor 71.0 12.9 11.8 Crucible Tree Armor chest armor 71.0 17.7 15.5 Crucible Axe Armor chest armor 71.0 17.5 15.5

In [0]:
%sql
SELECT armor_name, armor_type, resistance_focus, negation_magic, weight FROM eldenringcatalog.gold.fact_armor_stats ORDER BY resistance_focus DESC LIMIT 20;

armor_name armor_type resistance_focus negation_magic weight Corhyn's Robe chest armor 107.0 13.3 6.3 Braided Cord Robe chest armor 104.0 13.6 3.4 Azur's Glintstone Robe chest armor 99.0 15.4 7.1 Armor of Solitude chest armor 99.0 13.3 24.0 Mushroom Body chest armor 90.0 13.3 5.1 Armor of Solitude (Altered) chest armor 90.0 13.0 23.2 Astrologer Robe chest armor 90.0 13.5 6.3 Fire Prelate Armor chest armor 90.0 13.5 24.7 Preceptor's Long Gown chest armor 83.0 14.5 6.3 Prophet Robe chest armor 83.0 13.5 6.3 Page Garb chest armor 83.0 12.8 5.1 Sanguine Noble Robe chest armor 83.0 13.3 5.1 Lusat's Robe chest armor 83.0 15.4 7.1 Ansbach's Attire chest armor 76.0 13.0 5.1 Dryleaf Robe chest armor 76.0 14.1 5.1 Commoner's Garb chest armor 76.0 12.8 5.1 Noble's Traveling Garb chest armor 76.0 13.3 6.3 Traveling Maiden Robe chest armor 76.0 13.4 4.9 Commoner's Simple Garb chest armor 76.0 13.0 5.1 Millicent's Tunic chest armor 76.0 13.0 8.3

In [0]:
%sql
SELECT armor_name, armor_type, resistance_vitality, negation_holy, weight FROM eldenringcatalog.gold.fact_armor_stats ORDER BY resistance_vitality DESC LIMIT 20;

armor_name armor_type resistance_vitality negation_holy weight Fia's Robe chest armor 108.0 13.0 5.1 Deathbed Dress chest armor 107.0 12.4 3.2 Lusat's Robe chest armor 99.0 13.5 7.1 Fia's Robe (Altered) chest armor 93.0 12.6 3.2 Ansbach's Attire chest armor 90.0 13.0 5.1 Prophet Robe chest armor 90.0 13.0 6.3 Commoner's Simple Garb chest armor 83.0 12.8 5.1 Commoner's Garb chest armor 83.0 12.8 5.1 Noble's Traveling Garb chest armor 83.0 13.3 6.3 Azur's Glintstone Robe chest armor 83.0 13.8 7.1 Traveling Maiden Robe chest armor 83.0 13.4 4.9 Page Garb chest armor 83.0 13.0 5.1 Queen's Robe chest armor 83.0 13.3 5.1 Godskin Noble Robe chest armor 83.0 14.1 6.3 Fell Omen Cloak chest armor 83.0 12.8 5.1 Finger Robe chest armor 82.0 12.8 5.1 Depraved Perfumer Robe chest armor 80.0 12.7 5.0 Mushroom Body chest armor 76.0 13.0 5.1 Astrologer Robe chest armor 76.0 13.0 6.3 Millicent's Tunic chest armor 76.0 13.0 8.3

**The data shows that chest armor dominates all major resistance categories in Elden Ring, consistently outperforming other armor types in immunity, robustness, focus, vitality, and both physical and magical negation. Lightweight specialty pieces like the Mushroom Body, Braided Cord Robe, and Azur’s Glintstone Robe excel in immunity, focus, and vitality, offering strong defenses at very low weights. In contrast, heavy armors such as Verdigris Armor, Bull-Goat Armor, Tree Sentinel Armor, and Fire Prelate Armor provide top-tier robustness and physical negation but at the cost of much higher weight. Certain versatile pieces, like Lusat’s Robe and Fia’s Robe, balance high resistances across multiple categories with moderate weight, making them ideal for hybrid defensive builds. Overall, the dataset illustrates a clear trade-off: light chest pieces maximize efficiency and magical/poise defenses, while heavy chest armors prioritize raw physical and robustness protection.**

In [0]:
%sql
SELECT armor_name, armor_type,
    (negation_physical + negation_strike + negation_slash + negation_pierce + negation_magic + negation_fire + negation_lightning + negation_holy) AS total_negation,
    (resistance_immunity + resistance_robustness + resistance_focus + resistance_vitality + resistance_poise) AS total_resistance,
    (negation_physical + negation_strike + negation_slash + negation_pierce + negation_magic + negation_fire + negation_lightning + negation_holy) + 
    ((resistance_immunity + resistance_robustness + resistance_focus + resistance_vitality + resistance_poise) * 0.5) AS defense_score,
    weight
FROM eldenringcatalog.gold.fact_armor_stats
ORDER BY defense_score DESC LIMIT 30;

armor_name armor_type total_negation total_resistance defense_score weight Fire Prelate Armor chest armor 132.4 315.0 289.9 24.7 Armor of Solitude chest armor 127.3 317.0 285.8 24.0 Verdigris Armor chest armor 131.3 303.0 282.8 25.9 Bull-Goat Armor chest armor 131.7 302.0 282.7 26.5 Omen Armor chest armor 126.9 306.0 279.9 23.1 Gaius's Armor chest armor 126.89999999999999 298.0 275.9 20.1 Lionel's Armor chest armor 127.60000000000001 294.0 274.6 21.2 Armor of Solitude (Altered) chest armor 123.49999999999999 296.0 271.5 23.2 Tree Sentinel Armor chest armor 126.69999999999999 284.0 268.7 18.9 Veteran's Armor chest armor 125.6 283.0 267.1 18.9 Fire Prelate Armor (Altered) chest armor 127.39999999999999 271.0 262.9 23.6 Radahn's Lion Armor chest armor 123.19999999999999 273.0 259.7 17.5 Lionel's Armor (Altered) chest armor 124.39999999999998 268.0 258.4 20.2 Beast Champion Armor chest armor 123.89999999999999 261.0 254.39999999999998 17.5 Banished Knight Armor chest armor 123.7 257.0 252.2 17.5 Tree Sentinel Armor (Altered) chest armor 124.5 239.0 244.0 18.1 Veteran's Armor (Altered) chest armor 122.6 240.0 242.6 17.2 Scaled Armor chest armor 116.8 249.0 241.3 16.0 Malformed Dragon Armor chest armor 118.89999999999999 240.0 238.89999999999998 16.0 Radahn's Lion Armor (Altered) chest armor 120.8 233.0 237.3 16.4 Beast Champion Armor (Altered) chest armor 122.50000000000001 225.0 235.0 16.4 Banished Knight Armor (Altered) chest armor 120.6 220.0 230.6 16.5 Crucible Tree Armor chest armor 115.70000000000002 224.0 227.70000000000002 15.5 Crucible Axe Armor chest armor 115.7 224.0 227.7 15.5 Royal Knight Armor chest armor 117.5 211.0 223.0 15.5 Cleanrot Armor chest armor 111.8 217.0 220.3 15.0 Lusat's Robe chest armor 83.2 269.0 217.7 7.1 Azur's Glintstone Robe chest armor 81.6 272.0 217.6 7.1 Royal Knight Armor (Altered) chest armor 115.89999999999999 198.0 214.89999999999998 15.0 Mushroom Body chest armor 65.6 297.0 214.1 5.1

**The highest overall defense scores are dominated by heavy chest armors, with Fire Prelate Armor, Armor of Solitude, Verdigris Armor, and Bull-Goat Armor leading the list. These armors combine very high physical and elemental negation with strong overall resistances, giving them the top defense scores of around 280–290, though they come with substantial weight (20–26.5). Mid-tier chest armors like Omen Armor, Gaius’s Armor, and Lionel’s Armor also provide excellent all-around protection, balancing solid negation and resistance with slightly lower weight. Lightweight magical or specialty robes, such as Lusat’s Robe, Azur’s Glintstone Robe, and Mushroom Body, achieve respectable defense scores (~217) despite their low weight, highlighting their efficiency for mobility-focused builds. Overall, heavy chest armor maximizes total defense, while light mage-focused gear offers a highly weight-efficient alternative with decent protection.**

In [0]:
%sql
SELECT armor_name, weight, negation_physical, resistance_poise FROM eldenringcatalog.gold.fact_armor_stats WHERE armor_type = 'helm' ORDER BY weight LIMIT 3;

armor_name weight negation_physical resistance_poise Commoner's Headband (Altered) 0.7 0.1 1.0 Dancer's Hood 1.0 0.9 1.0 Leather Headband 1.0 1.4 2.0

In [0]:
%sql
SELECT armor_name, weight, negation_physical, resistance_poise FROM eldenringcatalog.gold.fact_armor_stats WHERE armor_type = 'helm' ORDER BY weight DESC LIMIT 3;

armor_name weight negation_physical resistance_poise Greatjar 12.3 8.0 14.0 Pumpkin Helm 12.3 7.0 11.0 Bull-Goat Helm 11.3 7.5 15.0

**The lightest helms are extremely mobility-friendly but offer minimal protection. For example, the Commoner's Headband (Altered) weighs only 0.7, providing almost no physical negation (0.1) and minimal poise (1). Similarly, Dancer’s Hood and Leather Headband are light (1 weight) but give only modest physical defense (0.9–1.4) and poise (1–2).**

**At the other end, heavy helms deliver substantial protection at a cost of mobility. The Greatjar and Pumpkin Helm weigh over 12, providing strong physical negation (7–8) and high poise (11–14), while the Bull-Goat Helm balances slightly less weight (11.3) with even higher poise (15) and solid physical defense (7.5).**

**In short, helm choice is a trade-off: light helms maximize speed and mobility with minimal defense, while heavy helms significantly boost physical protection and poise at the expense of weight.**

In [0]:
%sql
SELECT 
    CASE WHEN a.is_dlc = 1 THEN 'DLC' ELSE 'Base Game' END AS content_type,
    COUNT(*) AS armor_count,
    ROUND(AVG(s.weight), 2) AS avg_weight,
    ROUND(AVG(s.negation_physical), 2) AS avg_physical,
    ROUND(AVG(s.resistance_poise), 2) AS avg_poise
FROM eldenringcatalog.gold.dim_armors a
INNER JOIN eldenringcatalog.gold.fact_armor_stats s ON a.armor_name = s.armor_name
GROUP BY a.is_dlc;

content_type armor_count avg_weight avg_physical avg_poise DLC 145 6.02 6.09 10.82 Base Game 578 5.99 6.12 10.56

**Across Elden Ring, DLC and base game armors are very similar on average. DLC armors (145 pieces) have an average weight of 6.02, physical negation of 6.09, and poise of 10.82, while base game armors (578 pieces) average 5.99 weight, 6.12 physical negation, and 10.56 poise. This indicates that DLC armors maintain comparable defensive performance to base game gear, with only a slight increase in poise on average.**

In [0]:
%sql
SELECT armor_name, armor_type, weight,
       (negation_physical + negation_strike + negation_slash + negation_pierce +
        negation_magic + negation_fire + negation_lightning + negation_holy) AS total_negation,
       (resistance_immunity + resistance_robustness + resistance_focus + resistance_vitality + resistance_poise) AS total_resistance,
       ((negation_physical + negation_strike + negation_slash + negation_pierce +
         negation_magic + negation_fire + negation_lightning + negation_holy) + 
         (resistance_immunity + resistance_robustness + resistance_focus + resistance_vitality + resistance_poise)*0.5) AS defense_score
FROM eldenringcatalog.gold.fact_armor_stats
WHERE weight BETWEEN 10 AND 25
ORDER BY defense_score DESC
LIMIT 20;


armor_name armor_type weight total_negation total_resistance defense_score Fire Prelate Armor chest armor 24.7 132.4 315.0 289.9 Armor of Solitude chest armor 24.0 127.3 317.0 285.8 Omen Armor chest armor 23.1 126.9 306.0 279.9 Gaius's Armor chest armor 20.1 126.89999999999999 298.0 275.9 Lionel's Armor chest armor 21.2 127.60000000000001 294.0 274.6 Armor of Solitude (Altered) chest armor 23.2 123.49999999999999 296.0 271.5 Tree Sentinel Armor chest armor 18.9 126.69999999999999 284.0 268.7 Veteran's Armor chest armor 18.9 125.6 283.0 267.1 Fire Prelate Armor (Altered) chest armor 23.6 127.39999999999999 271.0 262.9 Radahn's Lion Armor chest armor 17.5 123.19999999999999 273.0 259.7 Lionel's Armor (Altered) chest armor 20.2 124.39999999999998 268.0 258.4 Beast Champion Armor chest armor 17.5 123.89999999999999 261.0 254.39999999999998 Banished Knight Armor chest armor 17.5 123.7 257.0 252.2 Tree Sentinel Armor (Altered) chest armor 18.1 124.5 239.0 244.0 Veteran's Armor (Altered) chest armor 17.2 122.6 240.0 242.6 Scaled Armor chest armor 16.0 116.8 249.0 241.3 Malformed Dragon Armor chest armor 16.0 118.89999999999999 240.0 238.89999999999998 Radahn's Lion Armor (Altered) chest armor 16.4 120.8 233.0 237.3 Beast Champion Armor (Altered) chest armor 16.4 122.50000000000001 225.0 235.0 Banished Knight Armor (Altered) chest armor 16.5 120.6 220.0 230.6

**the balanced armor sets with moderate weight between 10 and 25 units, prioritizing overall defense efficiency. The top entries are predominantly chest armor pieces, with Fire Prelate Armor leading with a defense score of 289.9, followed closely by Armor of Solitude (285.8) and Omen Armor (279.9). These armors combine high total negation against physical, elemental, and magical damage with substantial resistances in immunity, robustness, focus, vitality, and poise. Other notable armors include Gaius’s Armor, Lionel’s Armor, and Tree Sentinel Armor, all offering strong protection without excessively burdening the wearer. Overall, this selection demonstrates a careful balance between weight and defensive capabilities, making them ideal for players seeking robust protection while maintaining moderate mobility.**

In [0]:
%sql
SELECT armor_name, armor_type, weight, negation_physical, resistance_poise
FROM eldenringcatalog.gold.fact_armor_stats
WHERE weight <= 5
ORDER BY weight ASC, (negation_physical + resistance_poise) DESC
LIMIT 20;

armor_name armor_type weight negation_physical resistance_poise Commoner's Headband (Altered) helm 0.7 0.1 1.0 Gold Bracelets gauntlets 0.8 1.0 1.0 Braided Arm Wraps gauntlets 0.8 1.0 1.0 Brave's Bracer gauntlets 0.8 0.6 1.0 Circlet of Light helm 1.0 2.8 5.0 Leather Headband helm 1.0 1.4 2.0 St. Trina's Blossom helm 1.0 0.9 2.0 Brave's Cord Circlet helm 1.0 0.9 2.0 Dancer's Hood helm 1.0 0.9 1.0 Alberich's Pointed Hat (Altered) helm 1.0 0.9 1.0 Prophet Blindfold helm 1.0 0.2 1.0 Festive Hood (Altered) helm 1.0 0.2 1.0 Ascetic's Wrist Guards gauntlets 1.1 1.5 2.0 Battlemage Manchettes gauntlets 1.1 1.0 2.0 Lusat's Manchettes gauntlets 1.1 1.3 1.0 Azur's Manchettes gauntlets 1.1 1.0 1.0 Sorcerer Manchettes gauntlets 1.1 1.0 1.0 Millicent's Gloves gauntlets 1.1 1.0 1.0 Errant Sorcerer Manchettes gauntlets 1.1 0.6 1.0 Aristocrat Headband helm 1.2 1.9 2.0

**the lightweight armor pieces, ideal for players prioritizing mobility and speed while still retaining modest protection. The lightest items include Commoner’s Headband (Altered) at just 0.7 weight and gold or braided gauntlets at 0.8 weight, offering minimal physical negation and poise. Notably, Circlet of Light stands out with 2.8 physical negation and 5 poise at only 1 weight, providing a remarkable balance of protection and lightness. Other helms like Leather Headband, St. Trina’s Blossom, and Brave’s Cord Circlet provide modest stats while keeping the weight extremely low. Gauntlets like Ascetic’s Wrist Guards and Battlemage Manchettes also contribute with light protection without hindering movement. Overall, these armor pieces are best suited for agile or caster-focused builds that benefit from minimal encumbrance while still gaining some defensive value.**

In [0]:
%sql
SELECT armor_name, armor_type, weight, negation_physical, resistance_poise
FROM eldenringcatalog.gold.fact_armor_stats
WHERE weight >= 20
ORDER BY (negation_physical + resistance_poise) DESC
LIMIT 20;

armor_name armor_type weight negation_physical resistance_poise Bull-Goat Armor chest armor 26.5 20.4 47.0 Verdigris Armor chest armor 25.9 20.4 47.0 Fire Prelate Armor chest armor 24.7 19.2 45.0 Armor of Solitude chest armor 24.0 18.7 45.0 Fire Prelate Armor (Altered) chest armor 23.6 18.7 43.0 Omen Armor chest armor 23.1 18.3 43.0 Armor of Solitude (Altered) chest armor 23.2 18.3 43.0 Lionel's Armor chest armor 21.2 17.5 40.0 Lionel's Armor (Altered) chest armor 20.2 17.1 38.0 Gaius's Armor chest armor 20.1 16.8 38.0

**The heavy armor sets, which are designed for maximum protection and high poise, making them ideal for tanky, frontline playstyles. Leading the list are Bull-Goat Armor and Verdigris Armor, each weighing over 25 units while providing a whopping 20.4 physical negation and 47 poise, making them excellent for withstanding powerful enemy attacks. Other notable heavy sets include Fire Prelate Armor and Armor of Solitude, both offering nearly 19 physical negation and 45 poise at slightly lower weights around 24–24.7. Additional options like Omen Armor and Lionel's Armor provide strong defensive stats for slightly more maneuverable heavy builds. Overall, these armors are best suited for players who prioritize survivability over mobility, allowing them to endure massive damage while maintaining stability in combat.**

In [0]:
%sql
SELECT armor_name, armor_type, weight, negation_physical, negation_magic
FROM eldenringcatalog.gold.fact_armor_stats
ORDER BY negation_physical DESC, negation_magic DESC
LIMIT 20;

armor_name armor_type weight negation_physical negation_magic Verdigris Armor chest armor 25.9 20.4 13.5 Bull-Goat Armor chest armor 26.5 20.4 13.3 Fire Prelate Armor chest armor 24.7 19.2 13.5 Veteran's Armor chest armor 18.9 18.7 13.5 Radahn's Lion Armor chest armor 17.5 18.7 13.5 Banished Knight Armor chest armor 17.5 18.7 13.5 Armor of Solitude chest armor 24.0 18.7 13.3 Fire Prelate Armor (Altered) chest armor 23.6 18.7 13.3 Tree Sentinel Armor chest armor 18.9 18.7 13.0 Radahn's Lion Armor (Altered) chest armor 16.4 18.5 13.3 Tree Sentinel Armor (Altered) chest armor 18.1 18.5 12.6 Banished Knight Armor (Altered) chest armor 16.5 18.3 13.3 Armor of Solitude (Altered) chest armor 23.2 18.3 13.0 Omen Armor chest armor 23.1 18.3 13.0 Veteran's Armor (Altered) chest armor 17.2 18.1 13.1 Crucible Tree Armor chest armor 15.5 17.7 13.0 Lionel's Armor chest armor 21.2 17.5 13.5 Beast Champion Armor chest armor 17.5 17.5 13.0 Crucible Axe Armor chest armor 15.5 17.5 13.0 Beast Champion Armor (Altered) chest armor 16.4 17.2 12.9

**the most physically resilient armors in Elden Ring while also considering their magical resistance. At the top are heavy-hitting chest armors like Verdigris Armor and Bull-Goat Armor, both offering 20.4 physical negation and solid magic negation values of 13.3–13.5, making them highly reliable for tank-focused builds. Following closely are Fire Prelate Armor, Veteran’s Armor, and Radahn’s Lion Armor, providing slightly lower physical negation but maintaining strong magical defense, ideal for hybrid durability against both physical and elemental attacks. Other noteworthy armors, including Armor of Solitude, Tree Sentinel Armor, and Banished Knight Armor, balance high physical protection with moderate magic negation, giving players flexibility to withstand diverse combat threats. Overall, these armors cater to players prioritizing maximum physical resilience without sacrificing moderate magical protection, suitable for aggressive frontline roles or boss encounters.**

In [0]:
%sql
SELECT armor_name, armor_type, weight, negation_magic, negation_fire, negation_lightning
FROM eldenringcatalog.gold.fact_armor_stats
ORDER BY negation_magic DESC
LIMIT 20;

armor_name armor_type weight negation_magic negation_fire negation_lightning Lusat's Robe chest armor 7.1 15.4 12.8 13.3 Azur's Glintstone Robe chest armor 7.1 15.4 12.8 13.0 Preceptor's Long Gown chest armor 6.3 14.5 13.3 12.8 Royal Knight Armor chest armor 15.5 14.1 13.3 12.4 Dryleaf Robe chest armor 5.1 14.1 11.9 12.6 Gaius's Armor chest armor 20.1 14.1 13.8 13.8 Royal Knight Armor (Altered) chest armor 15.0 13.9 13.1 12.2 Queen's Robe chest armor 5.1 13.8 12.6 12.8 Dryleaf Robe (Altered) chest armor 4.1 13.8 11.4 12.4 Finger Maiden Robe chest armor 4.6 13.8 13.1 13.3 Lazuli Robe chest armor 4.1 13.8 11.9 12.4 Braided Cord Robe chest armor 3.4 13.6 11.9 12.9 Verdigris Armor chest armor 25.9 13.5 13.5 14.5 Prophet Robe chest armor 6.3 13.5 12.6 13.0 Radahn's Lion Armor chest armor 17.5 13.5 14.1 12.6 Astrologer Robe chest armor 6.3 13.5 13.0 13.3 Raya Lucarian Robe chest armor 4.1 13.5 12.6 12.4 Sage Robe chest armor 5.1 13.5 12.6 13.0 Fire Prelate Armor chest armor 24.7 13.5 19.8 13.0 Banished Knight Armor chest armor 17.5 13.5 13.5 13.0

**the armors with the highest magic defense in Elden Ring, ideal for caster-focused or hybrid builds. At the top are lightweight robes such as Lusat's Robe and Azur's Glintstone Robe, both offering 15.4 magic negation with moderate fire and lightning resistances, perfect for maximizing spell defense while keeping weight low. Close contenders include Preceptor's Long Gown and Dryleaf Robe, which combine high magic negation with decent elemental resistances. Heavy armors like Royal Knight Armor and Gaius's Armor provide slightly lower magic negation (13.9–14.1) but offer higher overall physical defense due to their weight and durability. Notably, Fire Prelate Armor stands out with 19.8 fire negation, making it a specialized choice against fire-heavy encounters. Overall, these armors cater to players who prioritize strong magical defense, with options ranging from light, agile sets to heavier, more protective variants, allowing flexibility for different combat scenarios.**

In [0]:
%sql
SELECT armor_name, armor_type, weight, resistance_poise, negation_physical
FROM eldenringcatalog.gold.fact_armor_stats
ORDER BY resistance_poise DESC, weight DESC
LIMIT 20;

armor_name armor_type weight resistance_poise negation_physical Bull-Goat Armor chest armor 26.5 47.0 20.4 Verdigris Armor chest armor 25.9 47.0 20.4 Fire Prelate Armor chest armor 24.7 45.0 19.2 Armor of Solitude chest armor 24.0 45.0 18.7 Fire Prelate Armor (Altered) chest armor 23.6 43.0 18.7 Armor of Solitude (Altered) chest armor 23.2 43.0 18.3 Omen Armor chest armor 23.1 43.0 18.3 Lionel's Armor chest armor 21.2 40.0 17.5 Tree Sentinel Armor chest armor 18.9 39.0 18.7 Lionel's Armor (Altered) chest armor 20.2 38.0 17.1 Gaius's Armor chest armor 20.1 38.0 16.8 Veteran's Armor chest armor 18.9 38.0 18.7 Tree Sentinel Armor (Altered) chest armor 18.1 35.0 18.5 Veteran's Armor (Altered) chest armor 17.2 35.0 18.1 Beast Champion Armor chest armor 17.5 34.0 17.5 Banished Knight Armor chest armor 17.5 34.0 18.7 Radahn's Lion Armor chest armor 17.5 34.0 18.7 Beast Champion Armor (Altered) chest armor 16.4 34.0 17.2 Banished Knight Armor (Altered) chest armor 16.5 33.0 18.3 Radahn's Lion Armor (Altered) chest armor 16.4 33.0 18.5

**the armors with the highest poise, making them ideal for players who want to tank hits and maintain stability during combat. Leading the list are heavysets like Bull-Goat Armor and Verdigris Armor, both with a poise of 47 and high physical negation (20.4), offering maximum resilience against stagger and physical attacks. Other strong contenders include Fire Prelate Armor and Armor of Solitude, which balance high poise (43–45) with substantial physical defense (18.3–19.2) while maintaining manageable weight for their class. Mid-weight armors such as Tree Sentinel Armor, Veteran’s Armor, and Lionel’s Armor provide a combination of good poise (33–39) and solid physical protection, suitable for players who want a compromise between mobility and defensive durability. Overall, these sets are optimal for tanking roles, heavy-hitters, or anyone prioritizing stability and stagger resistance in melee-heavy encounters.**

In [0]:
%sql
SELECT armor_name, armor_type, weight, negation_magic, resistance_poise
FROM eldenringcatalog.gold.fact_armor_stats
WHERE weight <= 5
ORDER BY negation_magic DESC, resistance_poise DESC
LIMIT 20;

armor_name armor_type weight negation_magic resistance_poise Finger Maiden Robe chest armor 4.6 13.8 10.0 Dryleaf Robe (Altered) chest armor 4.1 13.8 8.0 Lazuli Robe chest armor 4.1 13.8 8.0 Braided Cord Robe chest armor 3.4 13.6 5.0 Raya Lucarian Robe chest armor 4.1 13.5 7.0 Traveling Maiden Robe chest armor 4.9 13.4 10.0 Traveler's Clothes chest armor 4.1 13.3 8.0 Depraved Perfumer Robe chest armor 5.0 13.2 8.0 High Page Clothes (Altered) chest armor 4.3 13.2 8.0 Snow Witch Robe (Altered) chest armor 4.6 13.2 7.0 Traveling Maiden Robe (Altered) chest armor 4.3 13.1 8.0 Battlemage Robe chest armor 4.1 13.0 10.0 Finger Maiden Robe (Altered) chest armor 3.7 13.0 8.0 Errant Sorcerer Robe chest armor 4.1 13.0 8.0 Depraved Perfumer Robe (Altered) chest armor 4.0 13.0 7.0 Consort's Robe chest armor 4.1 13.0 7.0 High Priest Robe chest armor 4.2 12.8 10.0 Ansbach's Attire (Altered) chest armor 4.1 12.8 10.0 Brave's Battlewear chest armor 4.1 12.8 8.0 Upper-Class Robe chest armor 4.1 12.8 8.0

**The best light armors for players who want high magical protection without sacrificing mobility. The top choices are robes like Finger Maiden Robe, Dryleaf Robe (Altered), and Lazuli Robe, which weigh under 5 units and provide exceptional magic negation (13–13.8) while maintaining some poise (5–10). These sets are ideal for mages, hybrid spellcasters, or agile builds, allowing them to move quickly in combat while mitigating magic damage. Other notable options include Traveling Maiden Robe, Battlemage Robe, and High Priest Robe, offering a good balance of light weight, respectable poise, and strong magical defense. Overall, these armors are perfect for players prioritizing spell resistance, mobility, and light loadouts, making them highly effective in magical-heavy encounters or boss fights where dodging is key.**

In [0]:
%sql
SELECT armor_name, armor_type, weight,
       (negation_physical + negation_strike + negation_slash + negation_pierce +
        negation_magic + negation_fire + negation_lightning + negation_holy +
        resistance_immunity + resistance_robustness + resistance_focus + resistance_vitality + resistance_poise) AS total_defense
FROM eldenringcatalog.gold.fact_armor_stats
WHERE weight >= 20
ORDER BY total_defense DESC
LIMIT 20;

armor_name armor_type weight total_defense Fire Prelate Armor chest armor 24.7 447.4 Armor of Solitude chest armor 24.0 444.3 Verdigris Armor chest armor 25.9 434.3 Bull-Goat Armor chest armor 26.5 433.7 Omen Armor chest armor 23.1 432.9 Gaius's Armor chest armor 20.1 424.9 Lionel's Armor chest armor 21.2 421.6 Armor of Solitude (Altered) chest armor 23.2 419.5 Fire Prelate Armor (Altered) chest armor 23.6 398.4 Lionel's Armor (Altered) chest armor 20.2 392.4

**The heaviest and most defensive armor sets in Elden Ring, focusing on pieces weighing 20 units or more. Leading the list is Fire Prelate Armor (24.7 weight) with a total defense score of 447.4, followed closely by Armor of Solitude (444.3) and Verdigris Armor (434.3). These armors combine high physical and elemental negation with strong resistances across immunity, robustness, focus, vitality, and poise, making them ideal for players seeking maximum survivability in frontline combat. Other notable heavy sets include Bull-Goat Armor, Omen Armor, and Gaius's Armor, offering both durability and significant poise, which are critical for tanking damage-heavy bosses. These armors excel in heavy, high-defense builds, though their weight may reduce mobility, favoring a slow, resilient playstyle that trades speed for survivability.**

# **📊 10: Best poison/rot resistance (immunity)**

In [0]:
%sql
SELECT 
    armor_name,
    armor_type,
    resistance_immunity,
    negation_physical,
    weight
FROM eldenringcatalog.gold.fact_armor_stats
ORDER BY resistance_immunity DESC
LIMIT 20;

armor_name armor_type resistance_immunity negation_physical weight Mushroom Body chest armor 102.0 6.1 5.1 Verdigris Armor chest armor 82.0 20.4 25.9 Rotten Gravekeeper Cloak chest armor 76.0 6.7 6.3 Guardian Garb (Full Bloom) chest armor 71.0 10.2 8.8 Thiollier's Garb chest armor 71.0 8.8 7.1 Bull-Goat Armor chest armor 71.0 20.4 26.5 Shadow Militiaman Armor chest armor 69.0 7.1 7.1 Thiollier's Garb (Altered) chest armor 67.0 8.0 6.3 Tree Sentinel Armor chest armor 67.0 18.7 18.9 Gaius's Armor chest armor 67.0 16.8 20.1 Vulgar Militia Armor chest armor 67.0 8.8 7.7 Omenkiller Robe chest armor 67.0 8.8 7.1 White Reed Armor chest armor 67.0 9.5 8.3 Omen Armor chest armor 67.0 18.3 23.1 Fire Prelate Armor chest armor 67.0 19.2 24.7 Ronin's Armor chest armor 66.0 10.5 8.5 Depraved Perfumer Robe chest armor 65.0 6.0 5.0 Rotten Gravekeeper Cloak (Altered) chest armor 64.0 6.1 5.1 Chain-Draped Tabard chest armor 63.0 9.5 8.3 Cleanrot Armor chest armor 63.0 14.6 15.0

**The top armors in Elden Ring with the highest immunity resistance, which primarily helps reduce the effects of status ailments. Leading the list is the Mushroom Body (chest armor) with an immunity of 102, though its physical negation is moderate at 6.1 and it’s relatively light at 5.1 weight. Heavy hitters like Verdigris Armor (82 immunity, 20.4 physical negation, 25.9 weight) and Bull-Goat Armor (71 immunity, 20.4 physical negation, 26.5 weight) combine strong immunity with high physical protection, making them ideal for tanky builds. Other notable entries, such as Omen Armor and Fire Prelate Armor, provide balanced immunity and substantial negation, though they are heavier (over 20 weight). Light but high-immunity sets like Rotten Gravekeeper Cloak offer more mobility at the cost of lower physical defense. Overall, this list is essential for players who want status resistance prioritized while balancing weight and physical protection.**

# **📊 11: Best bleed/frost resistance (robustness)**

In [0]:
%sql
SELECT 
    armor_name,
    armor_type,
    resistance_robustness,
    negation_physical,
    weight
FROM eldenringcatalog.gold.fact_armor_stats
ORDER BY resistance_robustness DESC
LIMIT 20

armor_name armor_type resistance_robustness negation_physical weight Radahn's Lion Armor chest armor 99.0 18.7 17.5 Lionel's Armor chest armor 99.0 17.5 21.2 Verdigris Armor chest armor 97.0 20.4 25.9 Tree Sentinel Armor chest armor 90.0 18.7 18.9 Lionel's Armor (Altered) chest armor 90.0 17.1 20.2 Veteran's Armor chest armor 90.0 18.7 18.9 Beast Champion Armor chest armor 90.0 17.5 17.5 Bull-Goat Armor chest armor 83.0 20.4 26.5 Gaius's Armor chest armor 83.0 16.8 20.1 Scaled Armor chest armor 83.0 16.0 16.0 Banished Knight Armor chest armor 83.0 18.7 17.5 Radahn's Lion Armor (Altered) chest armor 82.0 18.5 16.4 Blaidd's Armor chest armor 76.0 14.6 13.7 Malformed Dragon Armor chest armor 76.0 16.8 16.0 Beast Champion Armor (Altered) chest armor 75.0 17.2 16.4 Tree Sentinel Armor (Altered) chest armor 75.0 18.5 18.1 Veteran's Armor (Altered) chest armor 75.0 18.1 17.2 Briar Armor chest armor 71.0 12.9 11.8 Crucible Tree Armor chest armor 71.0 17.7 15.5 Crucible Axe Armor chest armor 71.0 17.5 15.5

**This query lists the top 20 Elden Ring armors with the highest robustness, which mainly reduces staggering and knockback effects. Leading the pack are Radahn’s Lion Armor and Lionel’s Armor (both 99 robustness), combining high physical negation (18.7 & 17.5) with moderate-to-heavy weight (17.5 and 21.2). Verdigris Armor stands out for having very high physical negation (20.4) but comes with a heavy weight (25.9). Heavy tanky sets like Bull-Goat Armor and Banished Knight Armor offer a balance of robustness and physical protection, suitable for high-poise, frontline builds. Medium-weight options, like Scaled Armor and Malformed Dragon Armor, provide decent robustness with lower weight, making them better for mobility-focused tanks. Overall, the list helps players prioritize survivability against stagger while considering weight and physical defense.**

# **📊 Question 12: Best madness/sleep resistance (focus)**

In [0]:
%sql
SELECT 
    armor_name,
    armor_type,
    resistance_focus,
    negation_magic,
    weight
FROM eldenringcatalog.gold.fact_armor_stats
ORDER BY resistance_focus DESC
LIMIT 20;

armor_name armor_type resistance_focus negation_magic weight Corhyn's Robe chest armor 107.0 13.3 6.3 Braided Cord Robe chest armor 104.0 13.6 3.4 Azur's Glintstone Robe chest armor 99.0 15.4 7.1 Armor of Solitude chest armor 99.0 13.3 24.0 Mushroom Body chest armor 90.0 13.3 5.1 Armor of Solitude (Altered) chest armor 90.0 13.0 23.2 Astrologer Robe chest armor 90.0 13.5 6.3 Fire Prelate Armor chest armor 90.0 13.5 24.7 Preceptor's Long Gown chest armor 83.0 14.5 6.3 Prophet Robe chest armor 83.0 13.5 6.3 Page Garb chest armor 83.0 12.8 5.1 Sanguine Noble Robe chest armor 83.0 13.3 5.1 Lusat's Robe chest armor 83.0 15.4 7.1 Ansbach's Attire chest armor 76.0 13.0 5.1 Dryleaf Robe chest armor 76.0 14.1 5.1 Commoner's Garb chest armor 76.0 12.8 5.1 Noble's Traveling Garb chest armor 76.0 13.3 6.3 Traveling Maiden Robe chest armor 76.0 13.4 4.9 Commoner's Simple Garb chest armor 76.0 13.0 5.1 Millicent's Tunic chest armor 76.0 13.0 8.3

**The top armors by focus resistance in Elden Ring are mostly light to mid-weight robes, offering excellent protection against focus damage while maintaining mobility. Heavy options like Fire Prelate and Armor of Solitude provide high defense but reduce agility, making them ideal for tanky magic builds. Light armors such as Braided Cord, Lusat’s Robe, and Preceptor’s Long Gown favor rolling and fast casting, while balanced options like Astrologer Robe and Mushroom Body provide decent protection without sacrificing movement. Overall, players can choose between mobility-focused, tanky, or hybrid magic builds depending on weight and focus resistance priorities.**